site/docs/contribute.md

```diff
@@ -207,8 +218,8 @@ public class SnapshotManager implements ManageSnapshots {
   // existing code...
 
   @Override
-  public ManageSnapshots createBranch(String name, long snapshotId) {
```

Comment:
nit: Was this change intended? :)

Reply:
I can revert it, it seemed like a bug but will probably distract from the conversation.

Reply:
reverted.